frontend/src/pages/pipelines/global/experiments/artifacts/ArtifactDetails.tsx

```diff
@@ -0,0 +1,159 @@
+            <Truncate content={artifactName ?? 'Loading...'} />
+          </BreadcrumbItem>
+        </Breadcrumb>
+      }
+      empty={false}
+      provideChildrenPadding
+    >
+      <Tabs aria-label="Artifact details tabs" activeKey={ArtifactDetailsTabKey.Overview}>
+        <Tab
+          eventKey={ArtifactDetailsTabKey.Overview}
+          title={<TabTitleText>Overview</TabTitleText>}
+          aria-label="Overview"
+        >
+          <Flex
+            spaceItems={{ default: 'spaceItems2xl' }}
+            direction={{ default: 'column' }}
+            className="pf-v5-u-pt-lg pf-v5-u-pb-lg"
+          >
+            <FlexItem>
+              <Stack hasGutter>
+                <Title headingLevel="h3">Live system dataset</Title>
+                <DescriptionList isHorizontal data-testid="dataset-description-list">
+                  <DescriptionListGroup>
+                    {artifact?.uri && (
+                      <>
+                        <DescriptionListTerm>URI</DescriptionListTerm>
+                        <DescriptionListDescription>
+                          <ArtifactUriLink artifact={artifact} />
+                        </DescriptionListDescription>
+                      </>
+                    )}
+                  </DescriptionListGroup>
+                </DescriptionList>
+              </Stack>
+            </FlexItem>
+
+            {!!artifact?.propertiesMap.length && (
+              <FlexItem>
+                <Stack hasGutter>
+                  <Title headingLevel="h3">Properties</Title>
+                  <DescriptionList isHorizontal data-testid="props-description-list">
+                    <DescriptionListGroup>
+                      {artifact.propertiesMap.map(([propKey, propValue]) => (
+                        <React.Fragment key={propKey}>
+                          <DescriptionListTerm>{propKey}</DescriptionListTerm>
+                          <DescriptionListDescription>
+                            {propValue.stringValue}
+                          </DescriptionListDescription>
+                        </React.Fragment>
+                      ))}
+                    </DescriptionListGroup>
+                  </DescriptionList>
+                </Stack>
+              </FlexItem>
+            )}
+
+            {!!artifact?.customPropertiesMap.length && (
+              <FlexItem>
+                <Stack hasGutter>
+                  <Title headingLevel="h3">Custom properties</Title>
+                  <DescriptionList isHorizontal data-testid="custom-props-description-list">
+                    <DescriptionListGroup>
+                      {artifact.customPropertiesMap.map(([customPropKey, customPropValue]) => (
+                        <React.Fragment key={customPropKey}>
+                          <DescriptionListTerm>{customPropKey}</DescriptionListTerm>
+                          <DescriptionListDescription>
+                            {customPropValue.stringValue}
```

Comment:
The value is not always a string value, it could also be a number. For example, the accuracy here is 0.9 while it's not rendered because you cannot read anything from the string value.

<img width="1728" alt="Screenshot 2024-05-02 at 11 31 52 AM" src="https://github.com/opendatahub-io/odh-dashboard/assets/37624318/eab43972-9633-4900-bf66-e2889bb3bdda">

Here is the code snippet I got from the KF UI codebase about how they parse the value https://github.com/opendatahub-io/odh-dashboard/pull/2772/files#diff-074518a08242ffe4f7318325bd895ab814cebbf5b710a2beb10cadabe943eab5R14-R31, you can refer to it to get the value and render it correctly based on that like this
https://github.com/opendatahub-io/odh-dashboard/pull/2772/files#diff-ab1caf27cc6090cba8b4aaa13934c2db8b00e776f3df3a8eec675eca26a61880R13-R41

Reply:
This has been addressed.